src/nupic/os/Timer.cpp

```diff
@@ -66,38 +66,6 @@ static UInt64 getCurrentTime()
   return (UInt64)(v.QuadPart) - initialTicks_;
 }
 
-#elif defined(NTA_PLATFORM_darwin)
```

Comment:
> Done. @david-ragazzi - you can merge master and update it to use your new preprocessor flags

Thanks. Could someone merge it? (I asked to remove this permission to avoid make mistakes.)